### PR TITLE
Target on anchor does not work

### PR DESCRIPTION
**Pull request by FinalAngel**
We prevent links to go directly to the desired page to enable "double click to edit". The current implementation always redirects on the current page and is not taking ``target="_blank"`` or other attributes into account.

This PR solves the issue so ``<a href="http://www.google.ch" target="_blank">Google</a>`` will work.

### CODE REVIEW

File: cms/static/cms/js/modules/cms.plugins.js [hunk 1]
```diff
@@ -47,7 +48,9 @@ var CMS = window.CMS || {};
             if (++clickCounter === 1) {
                 timer = setTimeout(function () {
                     clickCounter = 0;
-                    window.location.href = $(e.currentTarget).attr('href');
+                    anchor = $(e.currentTarget);
```
Comment (vxsx) [inline cms/static/cms/js/modules/cms.plugins.js]: `var` should be here. otherwise we would hold the reference to the anchor forever

**Comment by vxsx** [comment]
not sure if there is no side effects when doing `window.open(url, '_self')`, but LGTM

File: cms/static/cms/js/modules/cms.plugins.js (hunk 1 above)
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.plugins.js]: I'm not sure, we do the same with timer so it might be better to also cache anchor, what you think?

**Comment by vxsx** [comment]
:+1:

**Comment by FinalAngel** [comment]
Me neither but thats the only way to trigger the target context. Eventually its handled as a pop up and will be blocked?

**Comment by vxsx** [comment]
unlikely, but if you do window.open inside of that new window it'll probably stick to that same window then

**Comment by vxsx** [comment]
:+1:

**Pull request merged**